apps/projects/app/components/Panel/NewProject/NewProject.js

```diff
@@ -113,110 +121,187 @@ const NewProject = () => {
-                        You’ll be able to prioritize your backlog, reach consensus on issue
-                        valuations, and allocate bounties to multiple issues.
-                    </Info>
-
-                    <Button
-                      mode="strong"
-                      wide
-                      onClick={handleNewProject}
-                      disabled={repoSelected < 0}
-                    >
-                      Submit
-                    </Button>
-                  </div>
-                )
-              }
-
-              if (loading) return (
-                <RepoInfo>
-                  <LoadingAnimation />
-                  <div>Loading repositories...</div>
-                </RepoInfo>
-              )
-
-              if (error) return (
-                <RepoInfo>
-                  <Text size="xsmall" style={{ margin: '20px 0' }}>
-                      Error {JSON.stringify(error)}
-                  </Text>
-                  <Button wide mode="strong" onClick={() => refetch()}>
-                      Try refetching?
-                  </Button>
-                </RepoInfo>
-              )
-            }}
-          </Query>
+  return (
+    <>
+      {github.status !== 'authenticated' ? (
+        <div style={{ width: '100%', textAlign: 'center', marginBottom: (3 * GU) + 'px' }}>
+          <Button
+            mode="normal"
+            onClick={handleGithubSignIn}
+            icon={<IconGitHub
+              color={`${theme.surfaceContentSecondary}`}
+              width="16px"
+              height="16px"
+            />}
+            label="Select from GitHub"
+          />
         </div>
-      </div>
-    </React.Fragment>
+      ) : (
+        <RepoQuery />
+      )}
+      <Button
+        mode="strong"
+        wide
+        onClick={handleNewProject}
+        disabled={repoSelected < 0}
+      >
+        Submit
+      </Button>
+    </>
+  )
+}
+GitHubRepoList.propTypes = PropTypes.func.isRequired
+
+const createProject = () => {
+  console.log('creating native project')
+}
+
+const NewProject = ({ handleGithubSignIn }) => {
+  const [ title, setTitle ] = useState('')
+  const [ description, setDescription ] = useState('')
+  const theme = useTheme()
+  const { appState: { github = { status: STATUS.INITIAL } } } = useAragonApi()
+
+  return (
+    <PanelContent>
+      <InfoBox>
+        Create a new project that belongs and operates entirely within this application, or synchronize one from GitHub.
+      </InfoBox>
+
+      <FormField
+        label="Title"
+        required={github.status !== 'authenticated' || title !== ''}
+        input={
+          <TextInput
+            wide
+            value={title}
+            onChange={e => setTitle(e.target.value)}
+            aria-label="Title"
+          />
+        }
+      />
+
+      {title !== '' ? (
+        <>
+        <FormField
+          label="Feedback"
```

Comment:
Wrong field name

Reply:
Amazing how so glaringly obvious can be missed. Fixed.